src/Microsoft.Framework.PackageManager/List/DependencyListCommand.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace Microsoft.Framework.PackageManager.List
+{
+    internal class DependencyListCommand
+    {
+        private readonly DependencyListOptions _options;
+
+        public static void Initialize(CommandLineApplication c, Reports reports)
+        {
+            c.Description = "Print the dependencies of a given project.";
+            var runtimeFolder = c.Option("--runtime",
+                "A path to the folder contains framework assemblies. When the command tries to resolve a framework project, such as KRE, there are assemblies lie in this folder. The command will try to find if there is an assembly in this folder bear the same name to resolve dependencies.",
```

Comment:
Need to make these help statements a bit clearer... "assemblies lie in this folder"?